plugins/nisq_analyzer.py

```diff
@@ -64,16 +94,47 @@ def get(self):
             version=plugin.version,
             type=PluginType.processing,
             entry_point=EntryPoint(
-                href=url_for(f"{NISQ_BLP.name}.PluginsView"),
+                href=url_for(f"{NISQ_BLP.name}.ProcessView"),
```

Comment:
add the expected query parameter to the ui href here (using <https://docs.python.org/3/library/urllib.parse.html#module-urllib.parse>)